### PR TITLE
User profile exists in db and backend

### PR DESCRIPTION
**Pull request by otakaran**
Code isn't the prettiest but it does what it needs to. Feedback welcome but probably won't fix until tomorrow

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const App = () => {
     // Check if user is logged in
     fire.auth().onAuthStateChanged((user) => {
       if (user) {
-        console.log("Holy shit we're logged in");
```
Comment (ethanchoi1) [inline src/App.js:19]: Very nice.

File: src/App.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const App = () => {
...
+        console.log("We're logged in");
```
Comment (ethanchoi1) [inline src/App.js:19]: good.

**Review by corimstyle — APPROVED** [review]
Looks good so far, I still would like general backend methods (getting a user, getting contacts, e.g.) but I can implement those

**Review by ethanchoi1 — APPROVED** [review]
Functionality for attempting to sign up to a duplicate/send message to a nonexistent user seems good to me. Great work!

**Comment by otakaran** [comment]
Forgot to report hours. Got stuck on working with async calls in js.
Time spent: 3.5 hours

**Pull request merged**